plugins/README.md

```diff
@@ -0,0 +1,28 @@
+## :plugins
+
+This folder holds all our Gradle- and Kotlin Compiler plugins:
+
+### [:plugins:dataframe-gradle-plugin](./dataframe-gradle-plugin)
+The Gradle plugin for DataFrame that can generate data schemas from a data sample using the simple
+task `dataframes {}`. It uses [:plugins:symbol-processor](./symbol-processor) to generate column accessors.
+
+### [:plugins:symbol-processor](./symbol-processor)
+The KSP plugin that can generate data schemas from a data sample using the `@file:ImportDataSchema` annotation.
+It is also used to detect (both manually written- or generated) `@DataSchema` annotated classes/interfaces to generate
```

Comment:
I'd rephrase to just "Generated extension properties for `DataSchema` annotated classes and interfaces, omit "detect" part